helpers/check_acme_routes.sh

```diff
@@ -18,9 +22,15 @@ COMMAND=${1:-"help"}
 # Set DRYRUN variable to true to run in dry-run mode
 DRYRUN="${DRYRUN:-"false"}"
 
+# Set DEBUG variable to true, to print echo messages
+DEBUG="${DEBUG:-"false"}"
+
 # Set a REGEX variable to filter the execution of the script
 REGEX=${REGEX:-".*"}
 
+# Set NOTIFYONLY to true if you want to only notify customers about their failed routes
```

Comment:
please describe what a "failed route" is, you don't use the term "failed" anywhere else in this script, so it's not really clear what this env variable really does.

Reply:
Changed the comment in:
```# Set NOTIFYONLY to true if you want to send customers a notification
 # explaining why Lagoon is not able to issue Let'S Encrypt certificate for
 # some routes defined in customer's .lagoon.yml file.
 # If set to true, no other action rather than notification is done (ie: no annotation or deletion)
```